CMakeLists.txt

```diff
@@ -1438,6 +1438,73 @@ if(WIN32)
   target_link_libraries(mixxx-lib PUBLIC FFTW::FFTW)
 endif()
 
+# libdjinterop (must be enabled explicitly to use)
+option(DJINTEROP "Use djinterop library" OFF)
+if(DJINTEROP)
+  message(STATUS "Enabling djinterop support...")
+  target_compile_definitions(mixxx-lib PUBLIC __DJINTEROP__)
+
+  # Look for an existing installation of libdjinterop and use that if available.
+  # Otherwise, download and build from GitHub.
+  find_package(DjInterop)
+  if(DjInterop_FOUND)
+    # An existing installation of djinterop is available.
+    message(STATUS "Linking dynamically to existing installation of libdjinterop")
+    target_include_directories(mixxx-lib PUBLIC ${DjInterop_INCLUDE_DIRS})
+    target_link_libraries(mixxx-lib PUBLIC DjInterop::DjInterop)
+  else()
+    # Fetch djinterop sources from GitHub and build them statically.
+    message(STATUS "Linking statically to libdjinterop fetched from GitHub")
+
+    # Meson is the preferred build system for libdjinterop, but may not be
+    # available to all developers wishing to merely compile Mixxx.  However,
+    # libdjinterop provides a minimal CMake build too for use with in-tree
+    # builds like this one, which we will default to.
+    option(DJINTEROP_MESON_BUILD "Build djinterop library with meson" OFF)
+
+    set(DJINTEROP_GIT_REPO_URL https://github.com/xsco/libdjinterop.git)
+    set(DJINTEROP_GIT_TAG tags/0.11.0)
+    set(DJINTEROP_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib/libdjinterop-install")
```

Comment:
If we drop support for meson, these can be specified inside the `ExternalProject_Add` call.

Reply:
Done.